style/style.css

```diff
@@ -0,0 +1,668 @@
+    width: 130px;
+    left: 205px;
+    top: 12px;
+    border-radius: 0;
+  }
+
+  .hero-container {
+    width: 100%;
+    height: 887px;
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    padding: 0 3%;
+    background-image: url(../images/mobile-background.png);
+  }
+
+  .hero-container-about {
+    width: 100%;
+    height: 723px;
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    padding: 0 3%;
+    background-image: url(../images/about_bg_main.png);
+  }
+
+  .auto-layout {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: auto;
+    background-color: #f7f7f8;
+  }
+}
+
+.campagin {
+  display: none;
+}
+
+@media (min-width: 992px) {
+  .campagin {
+    display: flex;
+    align-items: center;
+    color: #ec4327;
+    border: 4px solid #ec4327;
+    width: 125px;
+    padding: 4px 14px;
+    white-space: nowrap;
+  }
+
+  .hero-logo {
+    padding-left: 220px;
+    padding-top: 92px;
+  }
+
+  .navbar {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    padding: 0 198px 0 209px;
+    gap: 232px;
+    width: 100%;
+    height: 75px;
+    background: #fff;
+  }
+}
+
+.hero-logo {
+  text-align: center;
+}
+
+.mobile-logo {
+  padding-left: 3px;
+  padding-right: 3px;
+  display: block;
+}
+
+.desktop-logo {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  .mobile-logo {
+    display: none;
+  }
+
+  .desktop-logo {
+    display: block;
+  }
+
+  .sub-team {
+    box-sizing: border-box;
+    width: 757px !important;
+    height: 181px !important;
+    background: #f8f8f8;
+    border: 8.49673px solid #fff;
+    margin-top: 43px;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+    margin-left: 210px;
+    margin-right: 222px;
+    padding: 37px 30px;
+  }
+}
+
+.hero-container {
+  width: 100%;
+  min-height: 50vh;
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  padding: 0 3%;
+  background-image: url(../images/mobile-background.png);
+}
+
+.hero-container-about {
+  width: 100%;
+  min-height: 70vh;
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  padding: 0 3%;
+  background-image: url(../images/about_bg_main.png);
+}
+
+/* Media queries for responsiveness */
+@media screen and (max-width: 768px) {
+  .auto-layout {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 60px 15px;
+    gap: 10px;
+    height: auto;
+    background-color: #f7f7f8;
+  }
+
+  .sub-team p {
+    width: 100%;
+    height: 100%;
+    font-family: 'Lato', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 17px;
+    display: flex;
+    align-items: center;
+    letter-spacing: -0.01em;
+    color: #515151;
+    padding-top: 17px;
+    padding-right: 14px;
+    padding-left: 14px;
+  }
+}
+
+@media screen and (max-width: 480px) {
```

Comment:
- [ ] Hi, I noticed that your CSS styling is not built as mobile-first. You are using `@media (max-width:)` to adjust your layout for smaller screens, but this is not the best practice. Mobile-first design means that you start with the smallest screen size and then use `@media (min-width:)` to add more styles for larger screens. This will make your code more efficient and responsive, and also improve your performance and user experience. I suggest you refactor your CSS code to follow the mobile-first approach and see how it improves your design. You can read more about it here: https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Responsive/Mobile_first. Let me know if you have any questions! 😊